std/wasi/snapshot_preview1.ts

```diff
@@ -228,7 +228,8 @@ const clock_time_monotonic = function (): bigint {
 const clock_time_process = clock_time_monotonic;
 const clock_time_thread = clock_time_monotonic;
 
-function syscall(target: Function): Function {
+type SyscallFunction = (...args: unknown[]) => number;
```

Comment:
Syscall implies function so

```suggestion
type Syscall = (...args: unknown[]) => number;
```

Reply:
Thanks! I've renamed it.